### ISSUE TITLE
Nvda failed to read selected text in balabolka software

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9301]
### Steps to reproduce:
Install the balabolka software on the computer.
Open the software.
Please enter a text.
Press ctrl + shift + left arrow to select the word you just typed.
Press nvda + shift + s to read the word selection.
### Actual behavior:
When you select with ctrl + shift + left arrow the nvda is muted.
Pressing nvda + shift + s announces that there is no text selected.
When pressing delete see that the selected text has been deleted but nvda did not announce that it was selected.
While also working with the text area of the balaboka I noticed that the nvda fails from time to time saying that the line is empty, when it has text, pressing the arrows down or down again the nvda recovers and normally read the line. This second error does not seem to happen all the time and I was not able to get a place in a consistent space to get through.
### Expected behavior:
Nvda should announce that you are selecting the text, and pressing nvda + shift + s should announce the selected text.
It is also possible to correct the small flaws of reading empty lines where there are none.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
17134
#### Name and version of other software in use when reproducing the issue:
balabolka 2.15.0.687
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
log:
[balabouca.txt](https://github.com/nvaccess/nvda/files/2885881/balabouca.txt)

**Comment by DrSooom** [comment]
Dupblicate of issue #4756. And issue #8918 is related.

**Comment by ehollig** [comment]
Closing as duplicate of #4756
